com.unity.robotics.urdf-importer/Runtime/UrdfComponents/UrdfInertial.cs

```diff
@@ -134,12 +171,12 @@ private static Matrix3x3 ToMatrix3x3(Link.Inertial.Inertia inertia)
                                                                  (float)inertia.izz });
         }
 
-        private static Vector3 FixMinInertia(Vector3 vector3)
+        static Vector3 FixMinInertia(Vector3 vector3)
         {
-            for (int i = 0; i < 3; i++)
+            for (var i = 0; i < 3; i++)
```

Comment:
Why change it to var? :D

Reply:
To be consistent with Unity's style conventions: `var` is to be used in any place where type can be inferred. My IDE is set up to warn on style violations so I tend to make drive-by changes if I'm touching nearby code.

Reply:
TIL something new about our code conventions. Thanks